Makefile

```diff
@@ -0,0 +1,40 @@
+# Build Info Variables
+FILE=./VERSION
+RESUMIC_VERSION = `cat $(FILE)`
+GITCOMMIT = `git describe --tags --always --dirty`
```

Comment:
If we would print the command it would be better to execute the command before assigning to the variable.
`=` vs `:=` used in the assignment. See the difference below.

```
GITCOMMIT = `git describe --tags --always --dirty`
echo $GITCOMMIT -> `git describe --tags --always --dirty`
```
vs
```
GITCOMMIT := `git describe --tags --always --dirty`
echo $GITCOMMIT -> 4d9c21f-dirty
```

Reply:
Cool, didn't know this about Make! Should I change to `:=`? The commands are all silenced with `@`, so the output isn't seen anyways, but if we wanted to ever show the output this would make it cleaner.

Reply:
Up to you. If you feel you care more about the value on the variables `:=` might make more sense, but the result overall is the same for us currently.

Reply:
Cool. I'll leave it as-is for simplicity's sake, but this is a handy tip to know for the future! We might eventually want verbose debugging output (one possible scenario), so this is good to know.